Library/Homebrew/tap.rb

```diff
@@ -561,6 +567,14 @@ def formula_dir
   end
 
   # @private
+  def cask_dir
+    @cask_dir ||= begin
+      self.class.ensure_installed!
+      super
+    end
+  end
```

Comment:
Core tap, i.e.`homebrew/core` doesn't have casks. So you don't need this method.